src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -5140,6 +5243,24 @@ theorem sub_le_sub_iff_le {x y z : BitVec w} (hxz : z ≤ x) (hyz : z ≤ y) :
     BitVec.toNat_sub_of_le (by rw [BitVec.le_def]; omega)]
   omega
 
+theorem sdiv_neg_one {w : Nat} {x : BitVec w} :
+    x.toInt / -1 = - x.toInt := by
+  rcases w with _|w
+  · simp [of_length_zero]
+  · simp [toInt_allOnes]
```

Comment:
I wonder if this should be an `Int` theorem of the form `forall (x : Int), x / -1 = -x`. Let's do this in a subsequent PR even if we want to :)

Reply:
sure, happy to do that in a subsequent PR.